scss/detail.scss

```diff
@@ -1,57 +1,139 @@
-.detail__wrapper {
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
-  justify-content: center;
+.grid__wrapper {
+  display: grid;
+  grid-template: 1fr / 1fr;
+  height: 100%;
+  position: absolute;
+  width: 100%;
+}
+
+.grid__container {
+  align-self: center;
+  display: grid;
+  grid-template: 50px 1fr / 1fr;
+  height: 670px;
+  justify-self: center;
+  max-width: 870px;
+  padding: 32px;
+  width: 100%;
   
   @include for-mobile {
-    flex-grow: 0;
-    margin-bottom: 36px;
+    grid-template: 1fr 50px/ 1fr;
+    padding: 16px;
   }
 }
 
-.redirect {
-  display: flex;
-  width: 100%;
-  align-self: flex-start;
+.grid__redirect {
+  align-self: center;
+  justify-self: start;
+  
+  @include for-mobile {
+    align-self: center;
+    justify-self: center;
+    order: 1;
+  }
+}
+
+.redirect__button {
+  display: inline;
+  font-size: 20px;
+  
+  &::before {
+    content: url(assets/back-arrow-20.png);
+    margin-right: 12px;
+  }
+}
+
+.title {
+  font-size: 32px;
+  font-weight: 700;
+  grid-area: 1 / 1 / 2 / 3;
+  justify-self: center;
+
+  @include for-mobile {
+    font-size: 24px;
+  }
+}
 
-  &__button {
-    display: flex;
-    align-items: center;
+.subtitle {
+  align-self: center;
+  color: $gray;
+  font-size: 24px;
+  font-weight: bold;
+  grid-area: 2 / 1 / 3 / 3;
+  justify-self: center;
+  text-align: center;
+
+  @include for-mobile {
+    border-bottom: 4px solid $las-palmas;
+    font-size: 16px;
+    grid-area: 3 / 1 / 4 / 3;
+    line-height: 2.2;
+    width: 100%;
+  }
+}
+
+.info__items {
+  align-self: center;
+  grid-area: 4 / 2 / 5 / 3;
+  margin-left: 40px;
+  
+  @include for-mobile {
+    align-self: center;
+    grid-area: 4 / 1 / 5 / 3;
+    margin-left: 0;
+  }
+}
+
+.info__item {
+  &--main {
+    font-weight: bold;
+  }
+
+  &--secundary {
+    color: $gray;
+  }
+
+  &--main,
+  &--secundary {
     font-size: 20px;
-    
-    &::before {
-      content: url(assets/back-arrow-20.png);
-      margin-right: 12px;
+  }
+
+  @include for-mobile {
+    &--main,
+    &--secundary {
+      font-size: 14px;
     }
   }
+}
+
+.mb {
+  margin-bottom: 36px;
 
   @include for-mobile {
-    justify-content: center;
-    order: 1;
+    margin-bottom: 18px;
   }
 }
 
-.card {
+.grid__card {
   background-color: $white;
   box-shadow: 4px 4px 10px 0 rgba($black, 0.5);
-  display: flex;
-  max-width: 900px;
-  padding: 24px;
-  width: 100%;
-
+  display: grid;
+  grid-template: 40px 40px 1fr / 35% 1fr;
```

Comment:
I dont like that combine px fr and percentage, try to change percentage value

Reply:
ok